yt/data_objects/data_containers.py

```diff
@@ -174,11 +181,13 @@ def _set_center(self, center):
             elif center.lower() in ("max", "m"):
                 self.center = self.ds.find_max(("gas", "density"))[1]
             elif center.startswith("max_"):
-                self.center = self.ds.find_max(center[4:])[1]
+                field = self._first_matching_field(center[4:])
```

Comment:
It's good that you're making the de facto behaviour explicit here, could it be added to a docstring ?

Reply:
I would rather add that in a new PR, since the `sp = ds.sphere("maximum_density", [...])` syntax doesn't seem to even be in the doc.

Reply:
I have created a PR to keep track of this, as the function isn't even documented (see #3185).

Reply:
fine with me

Reply:
Yes! We may want to implement some simple ways of accessing field tuples only by their field name (either first-matching or discarding on-disks).